composer.json

```diff
@@ -23,17 +23,17 @@
         "php": "^7.1",
         "fzaninotto/faker": "^1.6",
         "myclabs/deep-copy": "^1.5.2",
-        "sebastian/comparator": "^3.0",	
-        "symfony/property-access": "^2.8 || ^3.4 || ^4.0",
-        "symfony/yaml": "^2.8 || ^3.4 || ^4.0"
+        "sebastian/comparator": "^3.0",
+        "symfony/property-access": "^2.8 || ^3.4 || ^4.0 || ^5.0",
```

Comment:
I'm starting to irk at the idea we are going as far back as 2.8; maybe in another PR it could be wise to drop PHP 7.1 for minimum PHP 7.2 and drop Symfony 2.x and 3.x as well

Reply:
I think 2.8 is being EOL'd at the end of the month, so that is probably a good idea yeah. I think 3.4 should still be supported though since it's LTS until end of 2021 and a lot of people will probably still be using it. (I know at my company we are).

Reply:
If it doesn't cost anything for supporting the 3.4 then cool otherwise it can be dropped: the old version will remain available